build/repo.targets

```diff
@@ -322,11 +322,17 @@
     <Error Text="Docker host must be using Linux containers." Condition="'$(DockerHostOS)' != 'linux'"/>
 
     <!-- Download runtime store archives -->
-    <Exec Command="dotnet restore --packages $(RuntimeStoreArchiveDirectory)" WorkingDirectory="$(RestoreRuntimeStoreDirectory)" />
+    <Message Text="Local runtime store signed artifacts not specified via RUNTIMESTORE_SIGNED_ARTIFACTS_PATH, falling back to artifacts availabe on myget." Importance="high" Condition="'$(RUNTIMESTORE_SIGNED_ARTIFACTS_PATH)' == ''"/>
+    <Exec Command="dotnet restore --packages $(RuntimeStoreArchiveDirectory)" WorkingDirectory="$(RestoreRuntimeStoreDirectory)" Condition="'$(RUNTIMESTORE_SIGNED_ARTIFACTS_PATH)' == ''"/>
```

Comment:
FYI, dotnet-restore can be written as an MSBuild command instead of using dotnet.exe.
```xml
<MSBuild 
    Projects="$(ToolsDir)tools\RestoreRuntimeStore\RestoreRuntimeStore.csproj"
    Properties="RestorePackagesPath=$(RuntimeStoreArchiveDirectory)" />
```